src/api/products/dell-streak-7.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
https://mate-academy.github.io/react_phone-catalog/api/products/dell-streak-7.json to fetch data